public/src/components/trading_terminal/OpenOrders.tsx

```diff
@@ -279,7 +280,12 @@ export class OpenOrders extends React.Component<OpenOrdersProps, OpenOrdersState
 					<td className="red volume"><strong>{openAsks[askId].getStockQuantity()}</strong></td>
 					<td className="red volume"><strong>{openAsks[askId].getStockQuantityFulfilled()}</strong></td>
 					<td className="red volume"><strong>{price}</strong></td>
+					{
+					 this.props.isMarketOpen == true ? 
```

Comment:
Use the `disabled` field of Semantic UI buttons instead of changing colour yourself.